jOOL-java-8/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -979,4 +979,694 @@ else if (percentile == 1.0)
             s -> s.map(Objects::toString).orElse("")
         );
     }
+
+    //CS304 Issue link: https://github.com/jOOQ/jOOL/issues/151
+
+    /**
+     * Calculate the linear regression function Slope of the given object collectors,
+     * based on the mapping function from object to double number.
+     *
+     * <p><pre> Usage of aggregation function regrSlopeBy():
+     * The mapping functions functionX and functionY are functions mapping the objects to double values X and Y, for instance:
+     * {@code
+     * FunctionX<Tuple2<Integer,Integer>, Double> mapping = e -> Double.valueOf(e.v1);
+     * FunctionY<Tuple2<Integer,Integer>, Double> mapping = e -> Double.valueOf(e.v2);
```

Comment:
We don't have such types `FunctionX` or `FunctionY`. Just call them `Function`

Reply:
If you want to clarify that these correspond to the `x, y` parameters of the linear regression function, you can still call the variables `mappingX` or just `x`

Reply:
I would like to use mappingX and mappingY, then.

Reply:
Let's skip this for now. I think we don't really need much Javadoc in a first step, to speed things up... I'll come up with some more standardised Javadoc examples later.